src/modules/Dashboard/features/RespondentData/RespondentDataReview/ReviewMenu/ReviewMenu.tsx

```diff
@@ -42,7 +42,7 @@ export const ReviewMenu = ({
           minDate={null}
           includeDates={responseDates}
           onMonthChange={onMonthChange}
-          disabled={false}
+          disabled={!responseDates?.length}
```

Comment:
I think it could be dangerous to disable the date picker this way, because the response days are set by this call:
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/e33e1fd2-9a09-4f47-8e3b-f5522dbae32e)
In my opinion, it's better to use the lastSeen date to disable the date picker if its value is null or undefined.
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/a2c1bb59-9ae3-48c3-b1d5-42b6f8b349d1)

Reply:
Good catch! updated